File: HelpDeskFrontend/src/app/bookmark.ts
// To parse this data:
//
//   import { Convert, Bookmark } from "./file";
//
//   const bookmark = Convert.toBookmark(json);

export interface Bookmark {
    id:       number;
    userId:   number;
    ticketId: number;
    ticket:   null;
    user:     null;
}

// Converts JSON strings to/from your types
export class Convert {
    public static toBookmark(json: string): Bookmark {
        return JSON.parse(json);
    }

    public static bookmarkToJson(value: Bookmark): string {
        return JSON.stringify(value);
    }
}

File: HelpDeskBackend/Models/Bookmark.cs
using System;
using System.Collections.Generic;

namespace HelpDeskBackend.Models;

public partial class Bookmark
{
    public int Id { get; set; }

    public int? UserId { get; set; }

    public int TicketId { get; set; }

    public virtual Ticket? Ticket { get; set; }

    public virtual User? User { get; set; }
}
